### PR TITLE
PA-645 adding ChesServiceCollectionTest

### PR DESCRIPTION
**Pull request by LawrenceLau2020**
testing the AddChesService

### CODE REVIEW

File: backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+namespace Pims.Dal.Test.Libraries.Ches
+{
+    [Trait("category", "unit")]
+    [Trait("category", "ches")]
+    [Trait("group", "notification")]
+    [ExcludeFromCodeCoverage]
+    public class ChesServiceCollectionTest
+    {
+       // public object Configuration { get; private set; }
```
Comment (Fosol) [inline backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs:26]: Remove line

File: backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+        public IConfiguration Configuration { get; }
+
+        #region Tests
+        #region ChesServiceCollection Constructors
```
Comment (Fosol) [inline backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs:31]: This isn't a test for `Constructors`

File: backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+
+            var builder = new ConfigurationBuilder();
+            var options = new ChesOptions()
+            {
+                AuthUrl = "a mocked value",
+                EmailEnabled = true,
+                EmailAuthorized = true
+            };
+            var chesJson = JsonSerializer.Serialize(new { Ches = options });
+            builder.AddJsonStream(new MemoryStream(Encoding.UTF8.GetBytes(chesJson)));
```
Comment (Fosol) [inline backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs:48]: Can you make sure this `MemoryStream` won't remain open after being used?  They're notorious for memory leaks.

**Review by Fosol — COMMENTED** [review]
Congrats on your 2nd PR!

File: backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs (hunk 3 above)
Comment (LawrenceLau2020) [reply backend/tests/unit/dal/Libraries/Ches/ChesServiceCollectionTest.cs:48]: What's the best way of disposing of this memory stream? The builder object doesn't have any methods to dispose of the MemoryStream object, would the garbage collection take care of it?

**Review by Fosol — APPROVED** [review]

**Review by devinleighsmith — APPROVED** [review]

**Pull request merged**